crates/rollup-boost/src/flashblocks/inbound.rs

```diff
@@ -17,13 +19,16 @@ impl FlashblocksReceiverService {
             url,
             sender,
             reconnect_ms,
+            metrics: Default::default(),
         }
     }
 
     pub async fn run(self) {
         loop {
             if let Err(e) = self.connect_and_handle().await {
                 error!("Flashblocks receiver connection error, retrying in 5 seconds: {e}");
+                self.metrics.reconnect_attempts.increment(1);
+                self.metrics.connection_status.set(0);
```

Comment:
this should be part of the health status instead of metrics?

Reply:
What do you mean? which health status?